clowdlet/cli.py

```diff
@@ -0,0 +1,74 @@
+"""CLI for Clowdlet. Contains the main logic of the module."""
+import argparse
+import sys
+from pathlib import Path
+
+import yaml
+
+from clowdlet.models.clowdapptemplate import ClowdAppTemplate
+
+
+def validate_args(args: argparse.Namespace) -> tuple[bool, str]:
+    """Check if all required args are present."""
+    if not args.file or not Path(args.file).is_file():
+        return False, "Expecting existing input file as an argument."
+    if not args.output or not Path(args.output).is_dir():
+        return False, "Expecting existing output directory as an argument."
+    if args.env_file and not Path(args.env_file).is_file():
+        return False, "Expecting existing env file as an argument."
+    return True, ""
+
+
+def main() -> None:
+    """Run the application."""
+    parser = argparse.ArgumentParser(prog="clowdlet")
+    parser.add_argument("-f", "--file", help="file with ClowdApp template")
+    parser.add_argument("-o", "--output", help="output directory where to put generated Quadlet files")
+    parser.add_argument("--env-file", help="file with custom env variables to render in the template")
+    args = parser.parse_args()
+    args_ok, detail = validate_args(args)
+
+    if not args_ok:
+        print(detail, file=sys.stderr)
+        sys.exit(1)
+
+    print(f"Loading ClowdAppTemplate file: {args.file}")
+    with Path(args.file).open() as template_file:
+        template_yaml = yaml.safe_load(template_file)
+
+    custom_env = {}
+    if args.env_file:
+        print(f"Loading custom env file: {args.env_file}")
+        with Path(args.env_file).open() as env_file:
+            for line in env_file:
+                parts = line.split("=", maxsplit=1)
```

Comment:
can it happen that variable in env file is written as `var = value` instead of `var=value`?